internal/request/graphql/schema/schema.go

```diff
@@ -77,15 +66,23 @@ func defaultMutationType() *gql.Object {
 	})
 }
 
-func defaultSubscriptionType() *gql.Object {
+func defaultSubscriptionType(fields ...*gql.Field) *gql.Object {
+	return defaultOperationType("Subscription", fields...)
+}
+
+func defaultOperationType(name string, fields ...*gql.Field) *gql.Object {
+	fieldsCfg := make(gql.Fields, len(fields)+1)
+	for _, field := range fields {
+		fieldsCfg[field.Name] = field
+	}
+	fieldsCfg["_"] = &gql.Field{
```

Comment:
question(all): We don't actually need these `_` query placeholders anymore, they are a very old artifact of how the GQL library works, but its no longer necessary, for either Subscription or Query.